prow/cluster/prow_job.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: extensions/v1beta1
+kind: ThirdPartyResource
```

Comment:
should probably switch to `CustomResourceDefinition` since TPRs are deprecated. I'd expect the upstream project to do the work fairly soon.